sunpy/net/jsoc/jsoc.py

```diff
@@ -522,13 +523,8 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
         dl_set = True
         if not downloader:
             dl_set = False
-            downloader = Downloader(progress=progress, overwrite=overwrite, max_conn=max_conn)
-
-        if downloader.max_conn * kwargs['max_splits'] > 10:
-            warn_user("JSOC does not support more than 10 parallel connections. " +
-                      f"Changing the number of parallel connections to {2 * self.default_max_conn}.")
-            kwargs['max_splits'] = 2
-            downloader.max_conn = self.default_max_conn
+            # Private communication from JSOC say we should not use more than one connection.
```

Comment:
Can we still log a message to `DEBUG` if a user is trying to pass a different `max_conn`?

Reply:
Something higher than that is probably justified, if someone tries to set a thing and we ignore it we should print a message which is visible by default (which means INFO or WARNING iirc).

Reply:
Fixed.